### DISCUSSION TITLE
Support of Ecoflow Smart Plug as Switch

### DISCUSSION THREAD

**Discussion by sybaron3** [tolwi/hassio-ecoflow-cloud#137] [Ideas]
Hi there,

Any plan to support EcoFlow Smart plug as sensor and switch within Home Assistant?

Great work for the Ecoflow Cloud integration,
Sylvain

**Comment by miklos73** [comment]
That would be really great, ideally with / including energy quantity (Wh, kWh)
Greetings from Switzerland!

**Reply by svenerbe** [reply]
I'm looking as well for smart plug integration including energy quantity (Wh, kWh)

**Reply by SymmaxX** [reply]
Ich auch

**Reply by andreasbuff** [reply]
Wäre super toll!

**Comment by xfirf** [comment]
Here are even some more examples:
https://github.com/Mark-Hicks/ecoflow-api/blob/main/examples/get_ef_api_sensor.py

Any chance we'll going to get this feature? :-)
